packages/button/src/LionButton.js

```diff
@@ -12,9 +12,13 @@ export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
         type: String,
         reflect: true,
       },
-      tabindex: {
-        type: Number,
+      // we us a property here as if we use the native tabIndex we can not set a default value
+      // in the constructor as it synchronously sets the attribute which is not allowed in the
```

Comment:
@gerjanvangeest can you extract this change to LionButton into a separate MR - pretty please 🤗 
probably will need some adoptions to tests as well (or if not tested then it should be)